graphs/possible_bipartition.py

```diff
@@ -8,5 +8,33 @@ def possible_bipartition(dislikes):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    pen_1 = []
+    pen_2 = []
 
+    for puppy in dislikes:
```

Comment:
Also note that by using a for-each loop you are using the entire list of dogs the `puppy` dislikes to represent each `puppy`/node. Consider how you might use the indices of the lists to represent each puppy/node instead.